geometry_msgs/Pose pose
float64 distance
bool execute
---
geometry_msgs/Pose awayPose
float64 planning_time
duration arrival_time